llvm/lib/Analysis/AliasAnalysis.cpp

```diff
@@ -327,6 +327,8 @@ AliasResult relaxSpecificAliasResult(const llvm::Value *Ptr1,
   WeakVH VH1 = AACache.findWeakVH(Ptr1);
   WeakVH VH2 = AACache.findWeakVH(Ptr2);
 
+  NumberOfAAQueries++;
+
```

Comment:
I also decided to move this around as it makes more sense to track here. If we track this here, we can compute the "cache hitrate" whereas before this was just at an arbitrary point.